xmlseq.go

```diff
@@ -370,6 +371,9 @@ func xmlSeqToMapParser(skey string, a []xml.Attr, p *xml.Decoder, r bool) (map[s
 				if len(na) > 0 {
 					n[skey] = na
 				} else {
+					if n == nil {
```

Comment:
Maybe do it above the `if` in line 371 above?

Reply:
Updated 👍